init.sql

```diff
@@ -924,6 +924,21 @@ BEGIN
             CONSTRAINT qsp_home_kpi_pkey PRIMARY KEY (kpi_id)
             )';
 
+        EXECUTE '
```

Comment:
how existing users get this table?

Reply:
Currently we don't have any automatic process for this. For now we can run DDL command for existing users.